src/Uno.UWP/ApplicationModel/PackageVersion.cs

```diff
@@ -22,5 +24,30 @@ internal PackageVersion(global::System.Version version)
 		public ushort Minor;
 		public ushort Build;
 		public ushort Revision;
+
+		public bool Equals(PackageVersion other)
+		{
+			return this == other;
```

Comment:
You should reverse the calls.

The linker/trimmer won't be able to remove `IEquatable<PackageVersion>.Equals(PackageVersion)` even if unused.

However it would be able to remove the `==` and/or `!=` operators... unless they are used (like in `Equals` here).

IOW it's more efficient to implement the comparison inside `Equals(PackageVersion)` and have
* `Equals(object)` call it;
* `==` call it;
* `!=` call it

Reply:
As mentioned in the comment above, I just copied whatever I found in the decompiled sources of `Microsoft.Windows.SDK.NET.dll` (10.0.22000.25), but I think we can safely diverge from Windows implementation?

Reply:
It would make sense to make the change to be linker friendly.